templates/kdump.conf.j2

```diff
@@ -18,3 +18,17 @@ path {{ kdump_path }}
 core_collector {{ kdump_core_collector }}
 {% endif %}
 default {{ kdump_system_action }}
+
+{% if ansible_facts['distribution'] in ['RedHat', 'CentOS'] %}
```

Comment:
What about Fedora?  If this applies to Fedora too, then you can use
```suggestion
{% if ansible_facts['os_family'] == 'Fedora' %}
```

Reply:
Fedora supports it starting from f35.

Reply:
ok - that's fine - we only support the current versions of Fedora